### PR TITLE
pritunl-client: init at 1.3.3300.95

### PR DESCRIPTION
**Pull request by lebzm**
###### Description of changes

[Pritunl](https://github.com/pritunl/pritunl) is a distributed enterprise vpn server built using the OpenVPN protocol. This PR adds [pritunl-client](https://github.com/pritunl/pritunl-client-electron/tree/master/cli) cli.

###### Things done

- Built on platform(s)
  - [x] x86_64-linux
  - [ ] aarch64-linux
  - [ ] x86_64-darwin
  - [ ] aarch64-darwin
- [ ] For non-Linux: Is `sandbox = true` set in `nix.conf`? (See [Nix manual](https://nixos.org/manual/nix/stable/command-ref/conf-file.html))
- [ ] Tested, as applicable:
  - [NixOS test(s)](https://nixos.org/manual/nixos/unstable/index.html#sec-nixos-tests) (look inside [nixos/tests](https://github.com/NixOS/nixpkgs/blob/master/nixos/tests))
  - and/or [package tests](https://nixos.org/manual/nixpkgs/unstable/#sec-package-tests)
  - or, for functions and "core" functionality, tests in [lib/tests](https://github.com/NixOS/nixpkgs/blob/master/lib/tests) or [pkgs/test](https://github.com/NixOS/nixpkgs/blob/master/pkgs/test)
  - made sure NixOS tests are [linked](https://nixos.org/manual/nixpkgs/unstable/#ssec-nixos-tests-linking) to the relevant packages
- [x] Tested compilation of all packages that depend on this change using `nix-shell -p nixpkgs-review --run "nixpkgs-review rev HEAD"`. Note: all changes have to be committed, also see [nixpkgs-review usage](https://github.com/Mic92/nixpkgs-review#usage)
- [x] Tested basic functionality of all binary files (usually in `./result/bin/`)
- [22.11 Release Notes (or backporting 22.05 Release notes)](https://github.com/NixOS/nixpkgs/blob/master/CONTRIBUTING.md#generating-2211-release-notes)
  - [ ] (Package updates) Added a release notes entry if the change is major or breaking
  - [ ] (Module updates) Added a release notes entry if the change is significant
  - [ ] (Module addition) Added a release notes entry if adding a new NixOS module
  - [ ] (Release notes changes) Ran `nixos/doc/manual/md-to-db.sh` to update generated release notes
- [x] Fits [CONTRIBUTING.md](https://github.com/NixOS/nixpkgs/blob/master/CONTRIBUTING.md).

### CODE REVIEW

File: pkgs/tools/networking/pritunl-client/default.nix [hunk 1]
```diff
@@ -0,0 +1,27 @@
...
+  modRoot = "cli";
+  vendorSha256 = "sha256-fI2RIzvfbqBgchsvY8hsiecXYItM2XX9h8oiP3zmfTA=";
+
+  postInstall = ''
+    mv $out/bin/cli $out/bin/pritunl-client
+  '';
+
+  meta = with lib; {
+    description = "Pritunl OpenVPN client CLI";
+    homepage = "https://github.com/pritunl/pritunl-client-electron/tree/master/cli";
```
Comment (AndersonTorres) [inline pkgs/tools/networking/pritunl-client/default.nix:23]:
```suggestion
    homepage = "https://github.com/pritunl/pritunl-client-electron/";
```

**Review by AndersonTorres — CHANGES_REQUESTED** [review]

File: pkgs/tools/networking/pritunl-client/default.nix (hunk 1 above)
Comment (bigzilla) [reply pkgs/tools/networking/pritunl-client/default.nix:23]: I thought it was the same as [[gopls](https://github.com/golang/tools/tree/master/gopls)](https://github.com/golang/tools/tree/master/gopls) where the repo is `tools` but the package is in subdirectory `gopls`.

https://github.com/NixOS/nixpkgs/blob/c5924154f000e6306030300592f4282949b2db6c/pkgs/development/tools/gopls/default.nix#L24

Comment (AndersonTorres) [reply pkgs/tools/networking/pritunl-client/default.nix:23]: Well, I never have seen such a thing before.
Monorepos containing many different programs I have seen, but not this specific code on Nixpkgs.
Ok then.

File: maintainers/maintainer-list.nix [hunk 2]
```diff
@@ -1745,6 +1745,12 @@
     githubId = 28444296;
     name = "Benjamin Hougland";
   };
+  bigzilla = {
```
Comment (AndersonTorres) [inline maintainers/maintainer-list.nix:1748]: `maintainers-list.nix` should be split on its own commit, with a message:

`maintainers: add bigzilla`

**Review by AndersonTorres — CHANGES_REQUESTED** [review]

File: pkgs/tools/networking/pritunl-client/default.nix (hunk 1 above)
Comment (bigzilla) [reply pkgs/tools/networking/pritunl-client/default.nix:23]: So can I resolve this? or should I still need to change it?

**Review by AndersonTorres — APPROVED** [review]
Given that it is not free, ofborg will not compile it. So I can't test it easily.

Otherwise LGTM

**Comment by bigzilla** [comment]
Hi, @AndersonTorres , may I ask a question? How long does it takes to get merge?

**Comment by AndersonTorres** [comment]
The time anyone remembers. I will merge it now.
